functions/Test-DbaSqlBuild.ps1

```diff
@@ -254,19 +254,21 @@ function Test-DbaSqlBuild {
                             $targetSP = 0
                         }
                         $targetSPName = $AllSPs[$targetSP]
-                        Write-Message -Level verbose -Message "Target SP is $targetSPName - $targetSP on $($AllSPs.Length)"
+                        Write-Message -Level Debug -Message "Target SP is $targetSPName - $targetSP on $($AllSPs.Length)"
```

Comment:
Needs to be `$($AllSPs.Length -1)`

Reply:
hum.... it's a debug message in "computer language", using 0-indexed numbers... in the end we want $AllSPs[$targetSP], so I don't see the point in changing the message

Reply:
The "problem" here is that RTM is being seen as "SP". That's why the output is 5.
For me the message `Target SP is SP4 - 4 on 5` indicates that I have an SP to apply and it's not true.
If we can have the message written in a way that doesn't offer doubts why not? 😄

Reply:
you need to look at the result, not at debug messages! `Target SP is SP4 - 4th element on 5 long array` would please you ?

Reply:
Not really. For that just leave it as it is right now.

Reply:
we can also do `Target SP is SP4` :-P